client/src/components/helpers/toggle-button.tsx

```diff
@@ -32,6 +32,7 @@ export default function ToggleButton({
 }: ButtonProps): JSX.Element {
   const checkIconStyle = {
     height: '15px',
+    paddingTop: '5',
```

Comment:
Is the Spacer not used in many more places than just this Component?

Reply:
I don't believe so, that's [this other spacer](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/components/helpers/Spacer.js). I only found this one spot anyway - maybe I'm mistaken, I only did a quick search.

Reply:
Quite right. It triggers my OCD a bit we have such similar components with the same name, though.